### ISSUE TITLE
WCAG 2.2 本体 : 5.2.5 非干渉 の注記 2

### ISSUE THREAD

**Issue by caztcha** [waic/wcag22#2170]
原文
> If a page cannot conform (for example, a conformance test page or an example page), it cannot be included in the scope of conformance or in a conformance claim.

現行訳
> ページが適合できない場合 (例えば、適合試験のページ又はサンプルページ)、そのページは適合の範囲又は適合表明の範囲に含めることはできない。

修正案
> 適合が不可能なページ (例えば、適合試験のページ又はサンプルページ) の場合、そのページを適合の範囲又は適合表明の範囲に含めることは不可能である。
